### PR TITLE
Document what wp db check doesn't do; provide suggestions for next commands

### PR DESCRIPTION
**Pull request by benlk**
## Changes

- adds documentation to note that `wp db check` doesn't check anything WordPress-specific
- adds suggested follow-up commands for checking WordPress-specific parts of the DB

## Why

#148

### CODE REVIEW

File: src/DB_Command.php [hunk 1]
```diff
@@ -258,6 +261,8 @@ public function check( $_, $assoc_args ) {
 
 		WP_CLI::debug( 'Associative arguments: ' . json_encode( $assoc_args ), 'db' );
 		WP_CLI::success( 'Database checked.' );
+		WP_CLI::log( 'Check whether the WordPress DB tables are installed: wp core is-installed', 'db' );
+		WP_CLI::log( 'Check whether the WordPress DB tables need to be updated: wp core update-db --dry-run', 'db' );
```
Comment (benlk) [inline src/DB_Command.php:265]: Do we want to wrap these in a conditional that checks for success from the `db check` command?

Comment (benlk) [reply src/DB_Command.php]: Updated at https://github.com/wp-cli/db-command/pull/244/commits/7923571db2fa2355174cf23358eed8859bd1cf07

File: src/DB_Command.php [hunk 2]
```diff
@@ -258,6 +261,8 @@ public function check( $_, $assoc_args ) {
 
 		WP_CLI::debug( 'Associative arguments: ' . json_encode( $assoc_args ), 'db' );
 		WP_CLI::success( 'Database checked.' );
+		WP_CLI::log( 'To check whether the WordPress DB tables are installed: wp core is-installed', 'db' );
+		WP_CLI::log( 'To check whether the WordPress DB tables need to be updated: wp core update-db --dry-run', 'db' );
```
Comment (danielbachhuber) [inline src/DB_Command.php:265]: We generally don't render any output after `WP_CLI::success()`.

For this particular problem, I'd prefer to remove these `WP_CLI::log()` statements and simply keep to the command doc changes.

**Review by danielbachhuber — CHANGES_REQUESTED** [review]

File: src/DB_Command.php (hunk 2 above)
Comment (benlk) [reply src/DB_Command.php]: Removed in https://github.com/wp-cli/db-command/pull/244/commits/1debf788f416b059d085b47ed5f4628c538495cf

**Review by danielbachhuber — APPROVED** [review]

**Pull request merged**